build/Dockerfile.helper

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18.0-alpine3.15
+FROM golang:1.20.3-alpine3.17
```

Comment:
not related to pr -it's just an internal image used in the pipeline.
i will push it to quay manually, and make sure it works.